.gitignore

```diff
@@ -40,3 +40,4 @@ custom_plan.rb
 pkg/
 config/hooks
 *.csv
+.envrc
```

Comment:
Install direnv, [add it to your shell's rc file](https://github.com/direnv/direnv#setup), then `echo "PATH_add bin" > .envrc` in the Foreman dir.

I'll add some details to the contribute webpage and to the -dev list if merged.